parent/pom.xml

```diff
@@ -934,6 +938,53 @@
       </dependencies>
     </profile>
 
+    <profile>
```

Comment:
What difference does this profile make? I compiled project on jdk11; apart from `ThorntailV2HealthCheckEnricherTest.java` project build fine. Perhaps I missed something?

Reply:
J11 removes some API from J10 (ie XML) so you need to add them manually. I cannot see the file but I guess this is what you mean in your comment.